review_wof/__init__.py

```diff
@@ -1,19 +1,20 @@
 """review_wof.py - Build report of patches and reviews
 """
+import yaml
 from flask import Flask, render_template
 
+from .model import patch_reviewers_view, patch_summary_view, patches_view
```

Comment:
I prefer it the way it was, and this is not related to the change you're proposing, so leave it like it is

Reply:
isort?

Reply:
well, i don't want to decide on a particulart style at this point, so you might as well disable it in your IDE for this project.

in any case you should not mix unrelated re-factoring and feature additions in the same PR.

if you do want to use isort - go ahead and run it on all the files and submit a PR with that, and perhaps also some CI action to verify it on new PRs.

Reply:
ack